includes/multi-currency/MultiCurrency.php

```diff
@@ -225,7 +225,7 @@ public function init_hooks() {
 		add_action( 'rest_api_init', [ $this, 'init_rest_api' ] );
 		add_action( 'widgets_init', [ $this, 'init_widgets' ] );
 
-		$is_frontend_request = ! is_admin() && ! defined( 'DOING_CRON' ) && ! WC()->is_rest_api_request();
+		$is_frontend_request = ! is_admin() && ! defined( 'DOING_CRON' ) && ! Utils::is_admin_api_request();
```

Comment:
@cesarcosta99 I think the idea here is to do the same from [FrontendPrices.php](https://github.com/Automattic/woocommerce-payments/blob/666d8c16bc2d792416aaba4629ad61ab5d81ce2f/includes/multi-currency/FrontendPrices.php#L49) and [FrontendCurrencies.php](https://github.com/Automattic/woocommerce-payments/blob/ddb568144eb78a48108eb846409b171a80be0565/includes/multi-currency/FrontendCurrencies.php#L110).